operator/option/config.go

```diff
@@ -109,10 +109,13 @@ const (
 	OperatorPrometheusServeAddr = "operator-prometheus-serve-addr"
 
 	// PProf enabled pprof debugging endpoint
-	PProf = "pprof"
+	PProf = "operator-pprof"
```

Comment:
Is there a way we could instead deprecate the old flag name for the operator and remove it in Cilium 1.14? Same for `pprof-port`/`operator-pprof-port`.

In any case we should also document the fact that the flag was rename in the version specific upgrade notes: https://docs.cilium.io/en/latest/operations/upgrade/#version-specific-notes

Reply:
Not really, no. The conflict means the old flag name will cause operator and agent to use the same port when reading from the configmap configfile, causing one to fail at startup. I originally avoided renaming the flag, but then the per-node config PR came up and I didn't want to have a special case here.

Reply:
Thanks for the explanation. I think this is fine given that these flags are presumably mainly used for development/debugging.